packages/xstate-immer/src/index.ts

```diff
@@ -61,3 +63,52 @@ export function createUpdater<TContext, TEvent extends ImmerUpdateEvent>(
     type
   };
 }
+
+const IMMER_PATCH_TYPE = '@immer.patch' as const;
```

Comment:
```suggestion
const IMMER_PATCH_TYPE = '@immer.patch';
```

`as const` is redundant here, as you are assigning a primitive value within a `const` declaration